examples/nodejs/cache/leaderboard.ts

```diff
@@ -0,0 +1,106 @@
+import {LeaderboardClient, LeaderboardConfigurations, CredentialProvider, SortedSetOrder} from '@gomomento/sdk';
+import {
+  LeaderboardDelete,
+  LeaderboardFetch,
+  LeaderboardGetRank,
+  LeaderboardLength,
+  LeaderboardRemoveElements,
+  LeaderboardUpsert,
+} from '@gomomento/sdk-core';
+
+async function main() {
+  const client = new LeaderboardClient({
+    configuration: LeaderboardConfigurations.Laptop.v1(),
+    credentialProvider: CredentialProvider.fromEnvironmentVariable({
+      environmentVariableName: 'MOMENTO_API_KEY',
+    }),
+  });
+
+  // upsert
+  const elements: Map<bigint, number> = new Map([
+    [123n, 100],
+    [456n, 200],
+    [789n, 300],
```

Comment:
nit: can we do `100.0` for score to indicate that we can also accept floating points as scores?